modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -103,6 +103,38 @@ public static double Acos(double s)
             return Math.Acos(s);
         }
 
+        /// <summary>
+        /// Returns the difference between the two angles,
+        /// in range of -<see cref="Pi"/>, <see cref="Pi"/>.
+        /// When <param name ="from"/> and <param name ="to"/> are opposite,
+        /// returns -<see cref="Pi"/> if <param name ="from/"> is smaller than <param name ="to"/>,
+        /// or <see cref="Pi"/> otherwise.
+        /// </summary>
+        /// <param name ="from">The start angle.</param>
+        /// <param name ="to">The destination angle.</param>
+        /// <returns>The difference between the two angles.</returns>
+        public static float AngleDifference(float from, float to)
```

Comment:
`from` is reserved in C# it seems

Reply:
Somehow when making the new branch, the "PI" from Math.PI and MathF.PI got removed? I don't know how that occured but i fixed it.